tools/gen_scenario_list.py

```diff
@@ -28,15 +28,18 @@
 FEATURE_DIRECTORY = "features"
 
 # list of subdirectories with feature files with specified order
-SUBDIRECTORIES = ("OCM",
-                  "OCP_WebConsole",
-                  "ACM",
-                  "Insights_Advisor",
-                  "Notification_Service",
-                  "SHA_Extractor",
-                  "insights-results-aggregator-cleaner",
-                  "insights-results-aggregator-exporter",
-                  )
+SUBDIRECTORIES = (
```

Comment:
if you don't want to maintain original order (and make harder to see history of changes in README.md) this is not needed

Reply:
do we want to maintain it?